src/tab.group.ts

```diff
@@ -159,12 +150,18 @@ export default class TabGroup extends LitElement {
 
   #resizeTimeout: ReturnType<typeof setTimeout> | null = null;
 
-  #selectedTab: Tab | null = null;
```

Comment:
I had to perform some surgery on Tab Group so I could understand what was going on. 

In doing so, I got rid of this field, which was out of date because it wasn't being set in `#onNavSlotChange()`. It's now a getter.